spec/models/fee/basic_fee_type_spec.rb

```diff
@@ -2,12 +2,37 @@
 
 module Fee
   describe BasicFeeType do
+
     let(:fee_type)  { build :basic_fee_type }
+    DATES_ATTENDED_APPLICABLE_FEES = %w( BAF DAF DAH DAJ PCM SAF )
+    DATES_ATTENDED_NOT_APPLICABLE_FEES = %w( CAV NDR NOC PPE NPW )
```

Comment:
It is deliberately separating the app logic from the test so as to raise an error in the event of a change to the app that is not accounted for in the test